queries.sql

```diff
@@ -80,3 +80,17 @@ SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE d
 
 -- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
 SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+SELECT a.name AS animal_name FROM animals AS a JOIN owners AS o ON a.owner_id = o.id WHERE o.full_name = 'Melody Pond';
+
+SELECT a.name AS animal_name FROM animals AS a JOIN species AS s ON a.species_id = s.id WHERE s.name = 'Pokemon';
+
+SELECT o.full_name AS owner_name, COALESCE(array_agg(a.name), '{}') AS owned_animals FROM owners AS o LEFT JOIN animals AS a ON o.id = a.owner_id GROUP BY o.full_name;
+
+SELECT s.name AS species_name, COUNT(a.id) AS animal_count FROM species AS s LEFT JOIN animals AS a ON s.id = a.species_id GROUP BY s.name;
+
+SELECT a.name AS digimon_name FROM animals AS a JOIN species AS s ON a.species_id = s.id JOIN owners AS o ON a.owner_id = o.id WHERE s.name = 'Digimon' AND o.full_name = 'Jennifer Orwell';
+
+SELECT a.name AS animal_name FROM animals AS a JOIN owners AS o ON a.owner_id = o.id WHERE o.full_name = 'Dean Winchester' AND a.escape_attempts = 0;
+
+SELECT o.full_name AS owner_name, COUNT(a.id) AS owned_animals_count FROM owners AS o LEFT JOIN animals AS a ON o.id = a.owner_id GROUP BY o.full_name ORDER BY owned_animals_count DESC LIMIT 1;
```

Comment:
- [ ] Nice work writing the SQL code for the queries :clap: However it would be nice to add the screenshots of the query results to your PR description. That will help to ensure that you run the queries with success and the results are correct.